src/webview/welcome/app/welcomePage.tsx

```diff
@@ -372,10 +372,10 @@ export class Welcome extends React.Component<DefaultProps, {
                         </div>
                         <section className='sticky-section-content'>
                             <div className='sticky-section-exten'>
-                                <p className='section__header-hint sticky-section-hint'>Allows developers to easily create, deploy and live debug container applications running on OpenShift &#38; Kubernetes. Thus enhancing the development inner loop workflow through One-click actions right from IDE.</p>
+                                <p className='section__header-hint sticky-section-hint'>Allows developers to easily <p className='highlight'>create, deploy and live debug</p> container applications running on OpenShift &#38; Kubernetes. Thus enhancing the development inner loop workflow through One-click actions right from IDE.</p>
```

Comment:
This 'highlight' class doesn't seem to change the text that's in the the `<p>` tag